egs/ami/s5/run_sdm.sh

```diff
@@ -10,17 +10,16 @@ mic=sdm$micid
 stage=0
 . utils/parse_options.sh
 
-# Set bash to 'debug' mode, it will exit on :
-# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
-set -e
-set -u
-set -o pipefail
-set -x
+# Set bash to 'debug' mode, it prints the commands (option '-x') and exits on : 
+# -e 'error', -u 'undefined variable', -o pipefail 'error in pipeline',
+set -euxo pipefail
 
 # Path where AMI gets downloaded (or where locally available):
-[ ! -r conf/ami_dir ] && echo "Please, run 'run_prepare_shared.sh' first!" && exit 1
-AMI_DIR=$(cat conf/ami_dir)
+AMI_DIR=$PWD/wav_db # Default,
```

Comment:
Unfortunately the domain-based locations are not applicable to the case when no domain is set like in the 'AWS' cluster. AFAIK, manually commenting/uncommenting or eventually adding a path is a simpler solution.